content/slack/index.md

```diff
@@ -0,0 +1,48 @@
+# Slack
+
+Slack is a group chat app that serves as our group's virtual space for all sorts of mumblechatter. You can access Slack from your phone, desktop, or browser.
+
+{{< button href="/schedule" >}}Join Slack{{< /button >}}
+</br>
+</br>
+
+## Channels
+
+Conversations on Slack are organized into channels that start with `#`. The following sections share details about the most common. Don't see one you want? Ask in `#mumblechatter`. You can even [create your own channel](https://slack.com/help/articles/201402297-Create-a-channel), such as for your Shield Lock or a one-time event!
+
+### Default channels
+
+Everyone automatically is part of the following channels:
+
+| Channel | Description | 
+| --- | --- |
+| #1stF | Anything fitness-related, including "preblasts" or promotional info for who's leading each workout the night before.|
+| #2ndF | Anything fellowship-related, such as happy hours, spontaneous meetups to watch a game, or recommendations on things to do.|
+| #3rdF | Anything faith-related, such as praises, prayers, and other ways we can help each other out.|
+| #backblasts | A channel where backblasts, or reports for each workout of the day, are automatically added based on the Q filling out the **Paxmate** bot. You can also review the [Paxmate Dashboard](/paxmate-dashboard/) based on these reports. |
```

Comment:
Thoughts on a separate channel for backblasts to reduce Slack clutter? It was one of the suggestions at an Exchange. I personally don't mind them in 1stF.

Reply:
It ends up being a great comment thread after each workout. Moving them will slide things around a bit more. It also with the mentions engages folks, but yeah not really sure. I'm adverse to changing too much, but we can certainly think about it.

Reply:
Will remove